src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5074,4 +5074,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_VoidInTuple" xml:space="preserve">
     <value>A tuple may not contain a value of type 'void'.</value>
   </data>
-</root>
+  <data name="ERR_NonTaskMainCantBeAsync" xml:space="preserve">
+    <value>Async Main methods must return Task or Task&lt;int&gt;</value>
```

Comment:
Changed.

---
In reply to: [114619543](https://github.com/dotnet/roslyn/pull/19093#discussion_r114619543) [](ancestors = 114619543,114608904)